src/sidebar/store/modules/selection.js

```diff
@@ -162,9 +163,12 @@ const update = {
   },
 
   ADD_ANNOTATIONS(state, action) {
-    const counts = tabs.counts(action.annotations);
+    const countNotes = arrayUtil.countIf(
```

Comment:
Wasn't sure if I should call the selector in the other store module here or just implement the counting here?

Reply:
I think implementing the counting here is fine, and a little more decoupled.